pkg/provider/configuration.go

```diff
@@ -383,7 +383,7 @@ func BuildTCPRouterConfiguration(ctx context.Context, configuration *dynamic.TCP
 			if len(configuration.Services) > 1 {
 				delete(configuration.Routers, routerName)
 				loggerRouter.
-					Error("Could not define the service name for the router: too many services")
+					Errorf("No service name for the router: %s, see https://doc.traefik.io/traefik/routing/providers/docker/#service-definition", routerName)
```

Comment:
I suggest to collect service names and print something like:
```suggestion
					Errorf("Service cannot be attached automatically  for router %q, router configuration should specify which of the many possible services should be attached: ", routerName, serviceNames])
```

Reply:
That can help user :+1: 
I tried to implement it with 42d28fd08
It's my first time using reflect, I'm unsure if it's the good way to get Keys from a map.